src/graphql/queries.js

```diff
@@ -433,11 +436,109 @@ export const getPrepaidCardBalanceQuery = gql`
   }
 `;
 
+export const getSubscriptionsQuery = gql`
+  query Collective($slug: String!) {
+    Collective(slug: $slug) {
+      id
+      type
+      slug
+      name
+      company
+      image
+      backgroundImage
+      description
+      twitterHandle
+      website
+      currency
+      settings
+      createdAt
+      stats {
+        id
+        balance
+        yearlyBudget
+        backers {
+          all
+          users
+          organizations
+          collectives
+        }
+        collectives {
+          host
+          parent
+        }
+        transactions
+        expenses {
+          id
+          all
+        }
+        totalAmountSent
+        totalAmountRaised
+      }
+      ordersFromCollective (subscriptionsOnly: true) {
```

Comment:
can you rename this to `orders` to be consistent with the naming convention?

Reply:
Then what would you call `orders` that belong to that Collective? These are orders fetched from a user or org's perspective.